src/details/transform.rs

```diff
@@ -104,28 +108,52 @@ const TRANSFORMS_INVERSE: [fn(Vec3, WhitePoint) -> Vec3; 14] = [
     ICtCp::ICtCp_PQ_to_RGB,
     //ICtCp_HLG,
     ICtCp::ICtCp_HLG_to_RGB,
+    //BT_601_709_2020,
+    bt601_oetf_inverse,
+    //PQ,
+    ST_2084_PQ_eotf,
 ];
 
-// Applies the sRGB "opto-eletric transfer function", i.e. gamma compensation
+/// Applies the sRGB OETF (opto-eletronic transfer function), sometimes called "gamma compensation"
 #[inline]
-pub fn sRGB_gamma(color: Vec3, _wp: WhitePoint) -> Vec3 {
+pub fn sRGB_oetf(color: Vec3, _wp: WhitePoint) -> Vec3 {
     let cutoff = color.cmplt(Vec3::splat(0.0031308));
     let higher = Vec3::splat(1.055) * color.powf(1.0 / 2.4) - Vec3::splat(0.055);
     let lower = color * Vec3::splat(12.92);
 
     Vec3::select(cutoff, lower, higher)
 }
 
-// Inverse of `color_gamma`
+/// Applies the sRGB EOTF (electro-optical transfer function), which is also the direct inverse of the sRGB OETF.
 #[inline]
-pub fn sRGB_gamma_inverse(color: Vec3, _wp: WhitePoint) -> Vec3 {
+pub fn sRGB_eotf(color: Vec3, _wp: WhitePoint) -> Vec3 {
     let cutoff = color.cmplt(Vec3::splat(0.04045));
     let higher = ((color + Vec3::splat(0.055)) / 1.055).powf(2.4);
     let lower = color / 12.92;
 
     Vec3::select(cutoff, lower, higher)
 }
 
+// Applies the BT.709/BT.601 OETF
+#[inline]
+pub fn bt601_oetf(color: Vec3, _wp: WhitePoint) -> Vec3 {
```

Comment:
Comment says BT.709/BT.601, function name is bt601, and the name of the TransformFn is BT_601_709_2020. Should make this more consistent I think, maybe using the first occurance (BT_601) as the name everywhere and describing the relationship to newer standards in comments?